tests/FSharp.Core.UnitTests/FSharp.Core/PrimTypes.fs

```diff
@@ -43,6 +43,24 @@ type LanguagePrimitivesModule() =
 
         let y = 2y
         Assert.AreEqual(y, y |> LanguagePrimitives.SByteWithMeasure<m> |> sbyte)
+        
+        let n = 2n
+        Assert.AreEqual(n, n |> LanguagePrimitives.IntPtrWithMeasure<m> |> nativeint)
```

Comment:
These would need to be guarded against preview somehow, @KevinRansom how do we do the testing for that with FSharp.Core APIs?

Reply:
I _think_ I've managed this already. But it was so easy that I'm a tad skeptical. :upside_down_face:

Reply:
FSharp.Core unit tests are compiled to test as LangeVersion Preview.  So this is fine.